modules/gke-cluster/variables.tf

```diff
@@ -34,57 +40,65 @@ variable "cluster_secondary_range_name" {
 
 variable "description" {
   description = "The description of the cluster"
+  type        = string
   default     = ""
 }
 
 variable "kubernetes_version" {
   description = "The Kubernetes version of the masters. If set to 'latest' it will pull latest available version in the selected region."
+  type        = string
   default     = "latest"
 }
 
 variable "logging_service" {
   description = "The logging service that the cluster should write logs to. Available options include logging.googleapis.com/kubernetes, logging.googleapis.com (legacy), and none"
+  type        = string
   default     = "logging.googleapis.com/kubernetes"
 }
 
 variable "monitoring_service" {
   description = "The monitoring service that the cluster should write metrics to. Automatically send metrics from pods in the cluster to the Stackdriver Monitoring API. VM metrics will be collected by Google Compute Engine regardless of this setting. Available options include monitoring.googleapis.com/kubernetes, monitoring.googleapis.com (legacy), and none"
+  type        = string
   default     = "monitoring.googleapis.com/kubernetes"
 }
 
 variable "horizontal_pod_autoscaling" {
   description = "Whether to enable the horizontal pod autoscaling addon"
+  type        = bool
   default     = true
 }
 
 variable "http_load_balancing" {
   description = "Whether to enable the http (L7) load balancing addon"
+  type        = bool
   default     = true
 }
 
 variable "enable_private_nodes" {
   description = "Control whether nodes have internal IP addresses only. If enabled, all nodes are given only RFC 1918 private addresses and communicate with the master via private networking."
+  type        = bool
   default     = "false"
```

Comment:
I am surprised this works. I would expect this to be a type error. In any case, probably should clean up because I expect this will eventually fail.

Reply:
@yorinasub17 should it be wrapped in quotes?

Reply:
Ah actually the opposite? remove the quotes!

Reply:
Fixed in https://github.com/gruntwork-io/terraform-google-gke/commit/7b8d4c1b839cf3fa64c51b5b9b287c9de0f57d59